index.js

```diff
@@ -19,6 +20,8 @@ const PASSTHROUGH_EVENTS = [
   'info_change'
 ]
 
+const COMPAT_SYMBOL = Symbol.for('org.interledgerjs.ilp-compat-plugin.idempotence')
```

Comment:
What is this for?

Reply:
If someone passes the same plugin to compat twice, we want to return the same wrapper instance. The best way is to add a property on the wrapped plugin that we can look at. The best way to ensure that that property doesn't collide with anything else is to use a Symbol.